tests/Fixer/PhpUnit/PhpUnitDataProviderReturnTypeFixerTest.php

```diff
@@ -193,7 +193,7 @@ public function provideFooCases() {}
     }
 
     /**
-     * @requires PHP ^7.4
+     * @requires PHP <8.0
```

Comment:
actually ^7.4 is more accurate

Reply:
- You can add 2 `@requires` and define min and max PHP version. PHPUnit's docs explicitly list supported operators (just like [`version_compare`](https://www.php.net/version_compare) used under the hood) and `^` is not there. Maybe it's undocumented PHPUnit's magic to resolve it in the background?
- yes, you _use_ "PHP ranges that allow for multiple PHP major versions implicitly", because all of our current `@requires 8.*` will be executed on PHP9 😉. Missing operator works as `>=`, just checked it minute ago to be sure (`@requires PHP 7.0` is executed on 8.3, while `@requires PHP ^7.0` is skipped)